TYPO3.Flow/Classes/TYPO3/Flow/Reflection/ReflectionService.php

```diff
@@ -671,6 +671,7 @@ public function isClassUnconfigurable($className)
      *
      * @param string $annotationClassName The annotation class name for a method annotation
      * @return array An array of class names
+     * @api
```

Comment:
I sneakily added this `@api` annotation because this should be allowed to be called from the "outside" IMO